### DISCUSSION TITLE
KaTeX with "mhchem extension"

### DISCUSSION THREAD

**Discussion by rsipakov** [timlrx/tailwind-nextjs-starter-blog#104] [Ideas]
Could you advise how can I set up the "[mhchem extension](https://github.com/KaTeX/KaTeX/tree/master/contrib/mhchem/)", if possible do it? Also, can you please tell me why it is not possible to successfully update "remark-math" to version 4.0.0? Thank you!

**Comment by timlrx** [comment]
Try adding it to the [document header](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/v1/pages/_document.js#L30)? The current master branch relies on `mdx/js` which uses `micromark` 12. `remark-math` 4 is meant for `micromark` 13 libraries. The v1 branch of the code uses a newer `xdm` parser that supports `remark-math` ^4.0.0 and the package dependencies have already been updated to include that.
